### PR TITLE
Made Mail and RtosTimer layout consistent

### PR DESCRIPTION
**Pull request by mbartling**
The rest of the document has been structured as 

## Thing

blurb 

## Thing class reference

Link

## Thing example

Link

### CODE REVIEW

File: docs/APIs/tasks/rtos.md [hunk 1]
```diff
@@ -132,16 +132,16 @@ This example shows ``Queue`` and ``MemoryPool`` (see below) managing measurement
...
 
 [![View code](https://www.mbed.com/embed/?url=https://developer.mbed.org/teams/mbed/code/rtos_mail/)](https://developer.mbed.org/teams/mbed/code/rtos_mail/file/tip/main.cpp)
 
-### Mail class reference
-
-[![View code](https://www.mbed.com/embed/?type=library)](https://docs.mbed.com/docs/mbed-os-api/en/mbed-os-5.3/api/classrtos_1_1Mail.html)
-
 ## RtosTimer
 
 <span class="warnings">**Warning**: DEPRECATED</br>
```
Comment (AnotherButler) [inline docs/APIs/tasks/rtos.md]: @iriark01 Please note that the RtosTimer goes away entirely in a later pull request, so you can ignore the "DEPRECATED" warning as long as we merge the other pull request.

**Comment by iriark01** [comment]
Darn. I went down the PRs instead of up, and I think I may have made a mess with this file.

**Comment by iriark01** [comment]
Thinking about it: if we only deprecated it in 5.3, it may not be a bad idea to leave it here with a deprecation message until 5.4 to help people find out what to replace it with.

**Comment by iriark01** [comment]
I agree with Austin. Leave it there for now with a deprecation message.

**Pull request merged**